volmdlr/faces.py

```diff
@@ -6982,6 +6982,41 @@ def plot(self, ax=None, color: str = 'k', alpha: float = 1):
 
         return ax
 
+    def project_coincident_faces_of(self, shell):
+        """
+        Divides self's faces based on coincident shell's faces
+        """
+
+        used_faces, list_faces = {}, []
+        initial_faces = self.faces[:]
+
+        for _, face1 in enumerate(initial_faces):
+            for _, face2 in enumerate(shell.faces):
```

Comment:
Here since you're not using the indexes from enumerate, could you remove them, please?

Reply:
I've added your commit ! Thanks !